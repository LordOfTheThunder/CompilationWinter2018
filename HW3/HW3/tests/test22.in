int potato(int a) {
	int t = a;
}
void tomato() {
	int x = potato();
}

void main() { print("Max is cool"); }
