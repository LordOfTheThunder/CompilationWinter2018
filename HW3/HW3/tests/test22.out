---end scope---
a INT -1
t INT 0
line 5: prototype mismatch, function potato expects arguments (INT)
